File: tests/integration/construct_component_slow/nodejs/component.ts
// Copyright 2016-2021, Pulumi Corporation.  All rights reserved.

import * as pulumi from "@pulumi/pulumi";

export class Component extends pulumi.ComponentResource {
    constructor(name: string, opts?: pulumi.ComponentResourceOptions) {
        super("testcomponent:index:Component", name, {}, opts, true);
    }
}

File: tests/integration/construct_component_slow/dotnet/Component.cs
// Copyright 2016-2021, Pulumi Corporation.  All rights reserved.

using Pulumi;

class Component : Pulumi.ComponentResource
{
    public Component(string name, ComponentResourceOptions opts = null)
        : base("testcomponent:index:Component", name, ResourceArgs.Empty, opts, remote: true)
    {
    }
}
